packages/editor/src/store/actions.js

```diff
@@ -812,9 +812,11 @@ export function setIsListViewOpened( isOpen ) {
  * Action that toggles Distraction free mode.
  * Distraction free mode expects there are no sidebars, as due to the
  * z-index values set, you can't close sidebars.
+ *
+ * @param {boolean} createNotice Whether to create a notice when this action is triggered.
  */
 export const toggleDistractionFree =
-	() =>
+	( createNotice = true ) =>
```

Comment:
```suggestion
	( { createNotice = true } = {} ) =>
```

Let's make this an object to be safer (in case of future additions) and have more clarity about the params we pass.